erpnext/hr/doctype/leave_application/test_leave_application.py

```diff
@@ -594,6 +598,18 @@ def set_leave_approver():
 	})
 	dept_doc.save(ignore_permissions=True)
 
+def make_email_notification():
+	if not frappe.db.exists("Email Template","Leave Status Notification"):
+		doc = frappe.get_doc(dict(
+			name = "Leave Status Notification",
+			subject = "Leave Status Notification",
+			response = "Dispatch Notification Test Response",
+			doctype = "Email Template"
+		))
+		doc.flags.ignore_validate=True
+		doc.flags.ignore_links=True
+		doc.insert()
```

Comment:
email notification testing is not required in leave application, you can remove this

Reply:
@sahil28297 i updated and remove hr setting for leave application notifications because on submit leave application that is throwing error.